tailcfg/tailcfg.go

```diff
@@ -2405,6 +2405,9 @@ const (
 	CapabilityDebug              NodeCapability = "https://tailscale.com/cap/debug"                 // exposes debug endpoints over the PeerAPI
 	CapabilityHTTPS              NodeCapability = "https"
 
+	// CapabilityMacUIV2 is used by macOS GUI clients to enabled/disable V2 UI features.
```

Comment:
typo "enable"

But this can't disable can it? It can only enable if present? Or is there some argument value that's used to disable?

How about:

   // CapabilityMacUIV2 makes the macOS GUI enable its v2 mode.